en/gps_compass/rtk_gps.md

```diff
@@ -144,9 +146,16 @@ The RTK GPS connection is essentially plug and play:
 
 ### Configuring GPS as Yaw/Heading Source
 
-GPS can be used as a source for yaw fusion when using modules where _yaw output is supported by the device_ (e.g. [Trimble MB-Two](../gps_compass/rtk_gps_trimble_mb_two.md)) or when using some [RTK GPS Setups with Dual u-blox F9P](../gps_compass/u-blox_f9p_heading.md).
+GPS can be used as a source for yaw fusion when using a single device with two antenna where _yaw output is supported by the device_, or when using some [RTK GPS Setups with Dual u-blox F9P](../gps_compass/u-blox_f9p_heading.md).
+Using GPS as a heading source has the benefit that yaw calculations are not impacted by magnetic interference.
```

Comment:
Any other benefits? I guess without a compass you don't need a mast. However you are susceptible to GNSS jamming.